specification/schemas/shops/Shop.json

```
@@ -125,6 +125,18 @@
                           "example": "We have an <b>awesome seasonal sample</b> sale in store next month! <a>Click here</a> to take a sneak peek.",
                           "maxLength": 250
                         },
+                        "customer_service_phone": {
+                          "type": [
+                            "string",
+                            "null"
+                          ],
+                          "description": "Phone number for customer service",
+                          "example": "+31 20 123 4567",
+                          "maxLength": 20
+                        },
+                        "customer_service_email": {
+                          "$ref": "#/components/schemas/Email"
+                        },
```

considering this field is nullable and to make it in line with the rest of the email content schema definitions, I think we should make this either schemas/Email, or null
```suggestion
                        "customer_service_email": {
                          "oneOf": [
                            {
                              "$ref": "#/components/schemas/Email"
                            },
                            {
                              "type": "null"
                            }
                          ]
                        },
```